src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  transition: all 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+    transition: 0.3s;
```

Comment:
The `transition` property is repeated in the `&:hover` selector. You can remove the `transition: 0.3s;` line as the `transition: all 0.3s;` is already defined in the `.card` class.